dist/angular-editable-text.js

```diff
@@ -24,14 +24,15 @@
         isEditing: '=ggIsEditing',
         showEllipsis: '=ggShowEllipsis',
         placeholder: '@',
+        autocomplete: '@',
         onChange: '&ggOnChange'
       },
       transclude: true,
       template:
         '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
           '<input class="no-animate" ng-show="!showEllipsis || (isEditing && showEllipsis)" ng-focus="onInputFocus()" ' +
             'ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
-            'ng-model="editingValue" placeholder="{{placeholder}}" type="text" ' +
+            'ng-model="editingValue" autocomplete="{{autocomplete}}" placeholder="{{placeholder}}" type="text" ' +
```

Comment:
It might be a good idea to use `ng-attr-autocomplete`, I am not sure how browsers would respond to an empty `autocomplete` attribute. It may be a moot point though if we are the only ones using the component

Reply:
It is just us using it - and hopefully for not much longer - but it's easy enough to fix. Fixed for autocomplete and placeholder attributes.